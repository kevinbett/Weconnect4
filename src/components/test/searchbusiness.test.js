import Search from '../SearchBusiness/index'
import { shallow } from "enzyme"

describe('search business page',() => { 
    it ('search business page renders', () => { 
        expect(shallow(<Search/>)).toMatchSnapshot()
    });
} )