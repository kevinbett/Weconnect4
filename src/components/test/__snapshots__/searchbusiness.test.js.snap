// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`search business page search business page renders 1`] = `
<div>
  <div
    className="row container"
  >
    <div
      className="col-md-4"
    >
      <form
        class="form"
      >
        <div
          class="input"
        >
          <div
            class=""
          >
            <input
              class="form-control"
              id="inlineFormInputGroup"
              onChange={[Function]}
              placeholder="Search..."
              type="text"
            />
          </div>
        </div>
      </form>
    </div>
    <div
      className="col-md-4"
    >
      <form
        class="form"
      >
        <div
          class="input"
        >
          <div
            class=""
          >
            <input
              class="form-control"
              id="inlineFormInputGroup"
              onChange={[Function]}
              placeholder="Location..."
              type="text"
            />
          </div>
        </div>
      </form>
    </div>
    <div
      className="col-md-4"
    >
      <form
        class="form"
      >
        <div
          class="input"
        >
          <div
            class=""
          >
            <input
              class="form-control"
              id="inlineFormInputGroup"
              onChange={[Function]}
              placeholder="category..."
              type="text"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
  <div
    className="row"
  >
    <SearchResults
      className="row"
      results={Array []}
    />
  </div>
</div>
`;
